packages/server/package.json

```
@@ -61,6 +61,7 @@
     "@smithy/util-stream": "3.3.2",
     "bcryptjs": "2.4.3",
     "body-parser": "1.20.3",
+    "bowser": "^2.11.0",
```

Can we make this an exact version?
```suggestion
    "bowser": "2.11.0",
```